Automattic/wp-calypso
# Calypso build: Transpile TypeScript

Ensure TypeScript files in packages are transpiled.

## Testing
- Verify 57d0a40362780238a84b3431bd7008ae07e1708b is OK. Try [Calypso-live](https://hash-57d0a40362780238a84b3431bd7008ae07e1708b.calypso.live). That includes packages with TypeScript UI components.
- Verify that .ts and .tsx files in packages are transpiled.
